e2e/integration/journey.test.js

```diff
@@ -7,6 +7,9 @@ beforeEach(() => {
   cy.intercept('GET', `${mockServerURL}/teams`, {
     fixture: 'teams.json'
   })
+  cy.intercept('GET', `${mockServerURL}/employers`, {
+    fixture: 'employers.json'
+  }).as('getEmployers')
```

Comment:
:bulb: If you don't assert on it you don't need the alias either.
```suggestion
  })
```

Reply:
Removed this alias